### ISSUE TITLE
how to disable only the RF part in ESP32-S2

### ISSUE THREAD

**Issue by edbek** [espressif/esp-idf#6802]
Help, how to disable only the RF part in ESP32-S2 ? This is to reduce the power consumption of the chip.
For this, can I use the command "esp_wifi_set_mode (WIFI_MODE_NULL);" ?

**Comment by EspHuifeng** [comment]
The RF part won't work if you don't initialize and start wifi.

**Comment by negativekelvin** [comment]
@EspHuifeng it is not clear if they want to disable it after using it or not use it at all. It would be good to know the expected power draw if the WiFi is initialized and inactive vs uninitialized.

**Comment by edbek** [comment]
I need to turn it off after initialization and then turn it on again.
Having rummaged a little in the codes, I have so far come to the conclusion that it is necessary to call the function to turn off the WiFi RF-part:

"esp_wifi_stop ()"

Is it so ? Can someone reliably answer this question?

**Comment by neg2led** [comment]
Yes, [`esp_wifi_stop();`](https://docs.espressif.com/projects/esp-idf/en/stable/esp32/api-reference/network/esp_wifi.html#_CPPv413esp_wifi_stopv) will shut off the RF block in an ESP32-S2. 

If you want to free as many resources as you can, [`esp_wifi_deinit();`](https://docs.espressif.com/projects/esp-idf/en/stable/esp32/api-reference/network/esp_wifi.html#_CPPv415esp_wifi_deinitv) will shut down the entire WiFi manager and free up the heap memory it uses.

Some more info can be found on the [Power Management](https://docs.espressif.com/projects/esp-idf/en/stable/esp32/api-reference/system/power_management.html#dynamic-frequency-scaling-and-peripheral-drivers) page

**Comment by edbek** [comment]
neg2led, Thank you !

**Comment by Alvin1Zhang** [comment]
Thanks for reporting and sharing the notes, feel free to reopen.
